tests/test_c8ql.py

```diff
@@ -10,7 +12,54 @@
     C8QLQueryKillError,
     C8QLQueryValidateError,
 )
-from tests.helpers import assert_raises, extract
+from tests.helpers import assert_raises, extract, generate_col_name
+
+
+def test_export_data_query(client, docs):
+    collection_name = generate_col_name()
+    client.create_collection(collection_name)
+    client.insert_document(collection_name=collection_name, document=docs)
+    resp = client.export_data_query(query='FOR u IN @@collection RETURN u',
+                                    bind_vars={'@collection': collection_name})
+    assert resp['error'] is False
+    assert resp['result'] == docs
+
+    random_value = random.randint(1, len(docs))
```

Comment:
Do we need this random_value? seems a bit unnecessary to me.. you can simply use some fixed value I guess for example:( ...'value': 1.. or use 'value': docs[0]["val"] )